tests/benchdnn/graph/input_displacer.cpp

```diff
@@ -229,29 +243,92 @@ partition_data_displacer_t::partition_data_displacer_t(
             } else if (aop.kind_ == "Select") {
                 if (select_cond_has_parent) {
                     // Implicit causal mask case.
-                    filling_type = filling_type_t::minus_infinity;
+                    filling_type = filling_type_t::fixed_setting;
+                    user_set_value = -INFINITY;
+                    cfg_name = "Implicit_causal_mask";
                 } else {
                     // Padding mask.
                     assert(qk_data_offset == 1 || qk_data_offset == 2);
                     // Fill condition depending on qk values tensor to use only
                     // its values, which is equivalent of not using a mask.
+                    filling_type = filling_type_t::fixed_setting;
                     if (qk_data_offset == 1) {
-                        filling_type = filling_type_t::one;
+                        user_set_value = 1.f;
                     } else if (qk_data_offset == 2) {
-                        filling_type = filling_type_t::zero;
+                        user_set_value = 0.f;
                     }
+                    cfg_name = "Explicit_padding_mask";
                 }
             }
 
             if (filling_type == filling_type_t::undef) {
                 BENCHDNN_PRINT(
                         7, "%s\n", "[DISPLACE]: Filling type was not set");
                 break;
+            } else if (filling_type == filling_type_t::fixed_setting) {
+                const std::vector<float> user_set {user_set_value};
```

Comment:
It would be better to fill a vector above instead of a single value and pass it directly to the `emplace`.